doc/md/guides/deploying/ecs-fargate.md

```diff
@@ -0,0 +1,174 @@
+guide, we will focus on the migration part only.
+
+Because of its operational simplicity, we will discuss deployment to ECS where tasks
+are run on Fargate, but the techniques discussed here are relevant to any ECS deployment.
+
+## Prerequisites
+
+Prerequisites to the guide:
+
+
+1. [A service running on ECS/Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_AWSCLI_Fargate.html)
+   defining your application.
+2. A connection string to a database that is accessible from the ECS task. (e.g. An RDS running in the same VPC as the
+   ECS cluster with a security group allowing access from the ECS task)
+3. Atlas and AWS CLIs installed and configured on your machine.
+
+## Storing database credentials in Secrets Manager
+
+In order to run migrations, Atlas needs a connection string to the database. In order to avoid storing the database
+credentials in plain text in the ECS task definition, we will use AWS Secrets Manager
+to store the database credentials and pass them to the migration container as environment
+variables.
+
+Let's start by creating a secret in AWS Secrets Manager that contains the database credentials:
+
+```bash
+aws secretsmanager create-secret --name mydb --secret-string 'postgres://user:password@host:port/dbname'
+```
+
+The CLI responds with the details about the created secret, which we will use later:
+
+```json
+{
+    "ARN": "arn:aws:secretsmanager:us-east-1:<account id>:secret:mydb-gxZ0Qe",
+    "Name": "mydb",
+    "VersionId": "ab6d1fc0-d1a0-49c8-9bfb-5fd9922ffc37"
+}
+```
+
+To make sure that the ECS task has access to the secrets, we will need to create add to the
+task's IAM role a policy that allows it to access the secrets. This will look something
+similar to:
+
+```json
+{
+   "Statement": [
+      {
+         "Action": [
+            "secretsmanager:GetSecretValue",
+            "secretsmanager:DescribeSecret"
+         ],
+         "Effect": "Allow",
+         "Resource": "arn:aws:secretsmanager:us-east-2:<account id>:secret:mydb-<random suffix>",
+         "Sid": ""
+      }
+   ],
+   "Version": "2012-10-17"
+}
+```
+
+## Reading secrets during deployment
+
+To read our secret value during deployment we can use the [`runtimevar`](https://atlasgo.io/atlas-schema/projects#data-source-runtimevar)
+data source. To use this, create a project file named `atlas.hcl`:
+
+```hcl 
+data "runtimevar" "url" {
+  url = "awssecretsmanager://mydb?region=us-east-2"
+}
+env "deploy" {
+  url = "${data.runtimevar.url}"
+}
+```
+
+Be sure to replace `mydb` with the name of your secret and to set the correct region in the query parameter.
+
+Next, create a Dockerfile that will include your migration directory and project file. This is a variation
+of the baseline example we introduced in the ["Creating container images for migrations"](image.md) guide:
+
+```dockerfile
+FROM arigaio/atlas:latest
+
+COPY migrations /migrations
+
+COPY atlas.hcl .
+```
```

Comment:
Is there anything like ConfigMap in ECS?

Reply:
as far as I can tell, no
https://www.reddit.com/r/aws/comments/9k10lp/ecs_question_providing_configs_to_a_container_at/
we can use parameter store..